OrbitGl/CaptureWindow.cpp

```diff
@@ -575,9 +575,9 @@ Batcher& CaptureWindow::GetBatcherById(BatcherId batcher_id) {
       return time_graph_.GetBatcher();
     case BatcherId::kUi:
       return ui_batcher_;
+    default:
+      UNREACHABLE();
```

Comment:
Interesting.. The downside here is that you won't get a compiler time error if someone adds new enum value, instead you will get a runtime error - maybe we should suppress this one?

Reply:
It's actually even more complicated. I tried to fix that a while ago. BatcherId is declared with a fixed underlying type uint32_t which means according to the standard that all possible uint32_t-values are valid, not only the ones with names. And the compiler won't work. The proper solution is not to use a fixed underlying type for that enum but that's complicated because it's used in a struct which is bitcast (or something similar - I don't remember from the top of my head.) So I gave up a while ago on this...

Reply:
Then maybe just remove the UNREACHABLE completely? But also fine, with keeping as it was before my change.

Reply:
But then at least we would get the runtime error. Henning, do you suggest reverting my change?

Reply:
No, I think it's the best we can do for now. Obviously a compile time check is better and a runtime-check just a last resort. So if clang-tidy suggests you to do it this way it's fine for me. I would change it though when we refactor BatcherId

Reply:
It was basically telling me, that UNREACHABLE is unreachable code :). So not 100% the obvious correct solution